Sources/TuistCache/Utilities/CacheFrameworkBuilder.swift

```diff
@@ -65,26 +65,31 @@ public final class CacheFrameworkBuilder: CacheArtifactBuilding {
 
     public func build(workspacePath: AbsolutePath,
                       target: Target,
+                      configuration: String?,
                       into outputDirectory: AbsolutePath) throws
     {
         try build(.workspace(workspacePath),
                   target: target,
+                  configuration: configuration,
                   into: outputDirectory)
     }
 
     public func build(projectPath: AbsolutePath,
                       target: Target,
+                      configuration: String?,
                       into outputDirectory: AbsolutePath) throws
     {
         try build(.project(projectPath),
                   target: target,
+                  configuration: configuration,
                   into: outputDirectory)
     }
 
     // MARK: - Fileprivate
 
     fileprivate func build(_ projectTarget: XcodeBuildTarget,
                            target: Target,
+                           configuration: String?,
```

Comment:
Eventually, `configuration` argument will be non-optional, because a `configuration` will always be provided via default config.

Reply:
Changed.